pkg/tinydtls/Makefile.include

```diff
@@ -2,6 +2,8 @@ PKG_BUILDDIR ?= $(PKGDIRBASE)/tinydtls
 
 INCLUDES += -I$(PKG_BUILDDIR)
 
+CFLAGS += -Wno-unused-parameter
```

Comment:
Does this have to be in Makefile.include? That way it gets applied globally. Maybe "Makefile" is a better choice?